eZ/Publish/Core/FieldType/TextBlock/SearchField.php

```diff
@@ -45,6 +45,18 @@ public function getIndexData(Field $field, FieldDefinition $fieldDefinition)
     }
 
     /**
+     * Extracts short snippet of the given $string.
+     *
+     * @param string $string
+     *
+     * @return string
+     */
+    private function extractShortText($string)
+    {
+        return substr(strtok(trim($string), "\r\n"), 0, 255);
```

Comment:
Good point, fixed in https://github.com/ezsystems/ezpublish-kernel/pull/1858/commits/1777d726e8e76f48369dd902d3d443abd9f2530e